docs/source/cli/create_connector.rst

```diff
@@ -0,0 +1,205 @@
+.. _cli.create_connector:
+
+===============
+Create Connector
+===============
+
+Description
+~~~~~~~~~~~
+
+The create connector command allows users to create a connector, including configuring IAM roles, mapping backend roles, and creating secrets automatically. Users can create a connector either interactively or by using a configuration file. Connectors can be created by simply selecting from a list of supported connectors and their associated models. The CLI then guides them through providing only the essential model-specific information, such as AWS region for Amazon services.
+
+Command Syntax
+~~~~~~~~~~~~~~
+
+``opensearch-ml connector create [--path <value>]``
+
+**Option:**
+
+* ``--path <value>``: Path to an existing connector configuration YAML file
+
+Usage Examples
+~~~~~~~~~~~~~~
+
+* Interactive connector creation:
+
+    Command:
+
+    ``opensearch-ml connector create``
+
+    Sample response:
+
+    .. code-block:: JSON
+
+        Starting connector creation...
+
+        Enter the path to your existing setup configuration file: /Documents/cli/setup_config.yml
```

Comment:
We need some prerequisite to create connector. Clarify this and link the setup document?